docs/blog/2018-12-19-kentico-cloud-and-gatsby-take-you-beyond-static-websites/index.md

```diff
@@ -4,7 +4,7 @@ date: 2019-01-07
 author: "Ondrej Polesny"
 image: "images/teaser.png"
 imageTitle: Kentico Cloud and Gatsby
-tags: ["react", "kentico cloud", "graphql"]
+tags: ["react", "graphql"]
```

Comment:
```suggestion
tags: ["react", "graphql", "content-mesh"]
```
Would "content-mesh" make sense to add in place of "kentico cloud" or even "sourcing"?